### PR TITLE
refactor(doc/contributing): improve descriptions/sections

### PR DESCRIPTION
**Pull request by PsychoShock**
This should be more clear for users now.

### CODE REVIEW

**Comment by PsychoShock** [comment]
I was thinking probably adding the tables method, just like https://github.com/citizenfx/natives/pull/977#pullrequestreview-1830851887. @4mmonium what do you think?

File: .github/CONTRIBUTING.md [hunk 1]
```diff
@@ -84,33 +83,37 @@ aliases: ["0x0123456789ABCDEF","_OLD_NATIVE_NAME"]
 ## _EXAMPLE_NATIVE
 
 ```c
-// 0x0123456789ABCDEF 0x1234ABCD
+// 0x3FEF770D40960D5A 0x1647F1CB
```
Comment (4mmonium) [inline .github/CONTRIBUTING.md:86]: I don't think changing this was necessary.

File: .github/CONTRIBUTING.md [hunk 2]
```diff
@@ -69,8 +69,7 @@ Submitting a pull request
 With over 5000 natives, it's hard for us to manually review and test all contributions. We love to see a well tested example to validate your change in the pull request description. Doing so will allow us to review your change much faster, and avoids untested or wrong information. This includes:
 
 - Code to test and validate your change with
-- Screenshots showcasing what the change is about
-- Source material of used information
```
Comment (4mmonium) [inline .github/CONTRIBUTING.md:73]: Source material of used information is needed, people should provide credit where it's due.

File: .github/CONTRIBUTING.md [hunk 3]
```diff
@@ -7,7 +7,7 @@ Contributions are greatly appreciated! Some topics to contribute on:
 - Documenting parameter and return types
 - Naming unnamed natives appropriately
 - Adding example code to natives
-- Reviewing existing pull requests
+- Reviewing existing pull requests / issues
```
Comment (4mmonium) [inline .github/CONTRIBUTING.md:10]: I believe this file was mainly created as a guide on how to create PRs.

**Review by 4mmonium — CHANGES_REQUESTED** [review]
Hey there, commented some concerns I had, thank you for your contribution!

File: .github/CONTRIBUTING.md (hunk 3 above)
Comment (PsychoShock) [reply .github/CONTRIBUTING.md:10]: Do you want me to remove `issues`? Would be great for people to know they can contribute answering the issues, no?

Comment (4mmonium) [reply .github/CONTRIBUTING.md:10]: Now that I think about it, you're correct. Since there's going to be more issues soon I believe.

**Comment by PsychoShock** [comment]
This should now be good @4mmonium 

I added the section `method signature` so people will know the standart from now.

**Comment by AvarianKnight** [comment]
I don't entirely see the point of this PR, this in my opinion, makes things harder to understand.

1. If we want an issue template we should specifically add one, burying it CONTRIBUTING.md seems like a bad idea.
2. Changing the examples at the bottom to improper format (Lua codegen will always be `UpperCamelCase` so randomly using `UPPER_SNAKE_CASE` is confusing)
3. Un-needed note about RedM native docs not being supported

Going to close this for now, some of this will hopefully get touched on later.

**Pull request closed without merging**